pandas/tests/frame/conftest.py

```diff
@@ -0,0 +1,121 @@
+import pytest
```

Comment:
A couple of things about this file:

* Do you use all of these fixtures in your changes?
* I would prefer if the naming is a little more consistent e.g.:
- You have the word "frame" is some of your fixture names but not othres
- You have underscores between words in some names but not others

Reply:
For this PR, I turned an often-used DF into a fixture - together with the other attributes of `TestData`. @jreback then told me to start a `conftest.py` and put it there, together with the other `TestData`-attributes used in this module (`frame`, `mixed_frame`).

I translated all the attributes into `conftest.py` *without* renaming them, so that they can be replaced on a per-module-basis as laid out in #22471. The names of the fixtures are clearly suboptimal, but following up on fixturizing the other modules would be much harder if I start renaming now.

Reply:
That's fair.  Let's save for a follow-up then.

Reply:
@gfyoung:

Since the names now changed after all, I thought I ask for your opinion on the naming/consistency.

Reply:
@h-vetinari : This looks great!

Reply:
@gfyoung 
Thanks. I just added a few more consistency fixes (but squashed to simplify reviewing), but only the first three fixture names were affected.